managers/models.py

```diff
@@ -90,3 +96,52 @@ def change_suspended_if_project_has_stop_date(sender: Project, **kwargs: Any) ->
             When(Q(stop_date__lt=timezone.datetime.now()), then=Value(False)), default=kwargs["instance"].suspended
         )
     )
+
+
+class ExpectedHoursQuerySet(models.QuerySet):
+    def get_expected_hours_sum_for_a_date(self, for_date: date, excluded_id: Optional[int] = None) -> timedelta:
```

Comment:
In my opinion, the name of the parameter `for_date` can be simply changed to `date`.

Reply:
Can't changed it. The change produces pylint error: Redefining name 'date' from outer scope.

Reply:
Changed